tower-web/src/app/modules/main/component/confirm-delete-dialog/confirm-delete-dialog.component.ts

```diff
@@ -0,0 +1,44 @@
+import { CommonModule } from '@angular/common';
+import { Component, Inject, NgModule } from '@angular/core';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialogModule, MatDialogRef, MatFormFieldModule, MatInputModule } from '@angular/material';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { Observable } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
+
+@Component({
+  selector: 'wt-confirm-delete-dialog',
+  templateUrl: './confirm-delete-dialog.component.html',
+  styleUrls: ['./confirm-delete-dialog.component.scss']
```

Comment:
Oh and I missed setting change detection to `OnPush` (I usually have it configured as the default strategy)